sys/linux/dev_vfio.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://www.kernel.org/doc/Documentation/vfio.txt
+# https://elixir.bootlin.com/linux/latest/source/drivers/vfio/vfio.c
+# https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/vfio.h
+# https://elixir.bootlin.com/linux/latest/source/drivers/vfio/vfio_iommu_type1.c
+
+include <uapi/linux/fcntl.h>
+include <linux/vfio.h>
+
+define VFIO_IOMMU_INFO_PGSIZES	(1 << 0)
```

Comment:
This is defined in vfio.h, so we should not need this. Please also check other defines.